### DISCUSSION TITLE
Improving the Circle Model Accuracy in 02 Notebook

### DISCUSSION THREAD

**Discussion by Hrithik2212** [mrdbourke/pytorch-deep-learning#81] [Ideas]
I got to 100 % Accuracy with Train set and 99% Accuracy with test for the  notebook 02_pytorch_classification (Binary Classification Problem) by changing the learning rate to 1.1

**Comment by mrdbourke** [comment]
Woah! Nice one @Hrithik2212, thank you for sharing :)
